src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -141,6 +131,74 @@ object ErrorAggregator {
       .agg(stats(0), stats.drop(1):_*)
       .coalesce(1)
   }
+  private def buildDimensions(meta: Meta): Row = {
+    val dimensions = new RowBuilder(dimensionsSchema)
+    dimensions("timestamp") = Some(new Timestamp(meta.Timestamp / 1000000))
+    dimensions("submission_date") = Some(new Date(meta.Timestamp / 1000000))
+    dimensions("channel") = Some(meta.normalizedChannel)
+    dimensions("version") = meta.`environment.build`.flatMap(_.version)
+    dimensions("build_id") = meta.`environment.build`.flatMap(_.buildId)
+    dimensions("application") = Some(meta.appName)
+    dimensions("os_name") = meta.`environment.system`.map(_.os.name)
+    dimensions("os_version") = meta.`environment.system`.map(_.os.version)
+    dimensions("architecture") = meta.`environment.build`.flatMap(_.architecture)
+    dimensions("country") = Some(meta.geoCountry)
+    dimensions("experiment_id") = for {
```

Comment:
Talked to @maurodoglio offline but for posterity, the docs for the new experiment block is here: https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/environment.html#experiments